### DISCUSSION TITLE
How to configure Python debugging in Code Server

### DISCUSSION THREAD

**Discussion by sakthivel-axim** [coder/code-server#5888] [General] [answered]
I have tried to install `ms python` extension for debugging but fails to install. 

The issue is the vs-code version in code-server doesn't support that extension. 

Any suggestion for python debugging would be helpful.

```
root@beaglebone:/home/debian# code-server --install-extension ms-python.python-2022.21.13431008.vsix 
Installing extensions...
Unable to install extension 'ms-python.python' as it is not compatible with VS Code '1.73.1'.
Failed Installing Extensions: file:///home/debian/ms-python.python-2022.21.13431008.vsix
```

Environment: 
OS - Debian (Beaglebone)
Code-Server - 4.9.0

**Comment by lg-nineoneninethreenine** [comment]
Damn this is seeen many times... Wonder how we could fix this...

Perhaps using an old version of the extension can fix this, try that and see if it works out.

**Reply by jsjoeio** [reply]
Yeah, can you try installing an older version? We need to upgrade to 1.74

**Comment by sakthivel-axim** [comment, marked as answer]
Great. Thanks @im-coder-lg , @jsjoeio. The older version of ms-python works.

```
root@beaglebone:/home/debian# code-server --install-extension ms-python.python-2022.18.2.vsix  
Installing extensions... 

Extension 'ms-python.python-2022.18.2.vsix' was successfully installed.
``` 

Still it supports latest version of extension will be helpful.

**Reply by lg-nineoneninethreenine** [reply]
Yeah that's a bummer. code-server, tbh is a bit slow when it comes to VS Code updates, but that's because other issues are there which have to be fixed. Once it look stable, the next release of VS Code is bundled I guess, since VS Code is an electron app and transpiling it to the Web needs some (possibly breaking) changes.
